### DISCUSSION TITLE
Filtering archived notes by tag on desktop

### DISCUSSION THREAD

**Discussion by SiGamma** [standardnotes/forum#2517] [Suggestions]
This feature is available on mobile, but there's no desktop equivalent. Smart tags don't allow for multiple note metadata fields to be checked (both tag field and archived status, for example), and I tried nesting views but it doesn't work. 

In general, is there a reason why filtering by tags works differently on desktop (radio buttons) and mobile (checkboxes)? Just curious if it's an accidental thing that stuck around or a conscious choice.

**Comment by atmoio** [comment]
The desktop app was built first, and the mobile app followed a few months later. So that gave us a few months to think things through. However, on mobile, the standard paradigm is one that makes multiple selection very easy and natural. On desktop, usually that requires cmd/ctrl + click. Not as straightforward. 

I think in the near future we will make this possible via the Folders extension.

The closest thing right now would be to choose your tag, then in the notes column, click Options, then toggle show "Archived Notes". This will display all archived notes in the list.
